test/mini_mime_test.rb

```diff
@@ -12,25 +12,25 @@ def test_that_it_has_a_version_number
   end
 
   def test_extension
-    assert_equal "application/zip", MiniMime.lookup_by_extension("zip").content_type
+    assert_equal "application/zip", MiniMime.lookup_by_extension("zip")&.content_type
   end
 
   def test_mixed_case
 
     # irb(main):009:0> MIME::Types.type_for("TxT").first.to_s
     # => "text/plain"
 
-    assert_equal "application/vnd.groove-tool-message", MiniMime.lookup_by_filename("a.GTM").content_type
-    assert_equal "application/zip", MiniMime.lookup_by_extension("ZiP").content_type
+    assert_equal "application/vnd.groove-tool-message", MiniMime.lookup_by_filename("a.GTM")&.content_type
+    assert_equal "application/zip", MiniMime.lookup_by_extension("ZiP")&.content_type
```

Comment:
hmmm I don't understand these changes ...

Reply:
Just making sure that if the lookup fails and returns `nil` we get a clean failure from the `assert_equals` instead of a NoMethodError from calling `.content_type` on `nil`. I think I ran into that while testing my code.